core/src/main/java/com/graphhopper/storage/GraphHopperStorage.java

```diff
@@ -65,22 +65,7 @@ public GraphHopperStorage(Directory dir, EncodingManager encodingManager, boolea
         this.dir = dir;
         this.properties = new StorableProperties(dir);
         this.segmentSize = segmentSize;
-        InternalGraphEventListener listener = new InternalGraphEventListener() {
-            @Override
-            public void initStorage() {
-                for (CHGraphImpl cg : chGraphs) {
-                    cg.initStorage();
-                }
-            }
-
-            @Override
-            public void freeze() {
-                for (CHGraphImpl cg : chGraphs) {
-                    cg._prepareForContraction();
```

Comment:
No need for this there-and-back loop anymore. Right now we call freeze in GraphHopperStorage, which calls freeze in BaseGraph, which then executes the listener it got in the constructor earlier, and the listener calls `prepareForContraction`. But GraphHopperStorage can just as well simply call `prepareForContraction` directly. This is probably left over from some previous refactorings.